dashboard/db/migrate/20160219021859_add_submitted_to_user_levels.rb

```diff
@@ -0,0 +1,14 @@
+class AddSubmittedToUserLevels < ActiveRecord::Migration
+  def change
+    add_column :user_levels, :submitted, :boolean
+
+    reversible do |change|
+      change.up do
+        execute('UPDATE user_levels SET submitted = true, best_result = 30 WHERE best_result = 1000')
```

Comment:
Are you sure? Using execute which calls SQL directly (instead of update or save through a model object) should leave the timestamps untouched (you can verify this on your dev db if worried).


> On Feb 19, 2016, at 12:06, Brendan Reville <notifications@github.com> wrote:
> 
> In dashboard/db/migrate/20160219021859_add_submitted_to_user_levels.rb:
> 
> > @@ -0,0 +1,14 @@
> > +class AddSubmittedToUserLevels < ActiveRecord::Migration
> > +  def change
> > +    add_column :user_levels, :submitted, :boolean
> > +
> > +    reversible do |change|
> > +      change.up do
> > +        execute('UPDATE user_levels SET submitted = true, best_result = 30 WHERE best_result = 1000')
> Good point. Apparently it's possible to not touch these: http://stackoverflow.com/questions/861448/is-there-a-way-to-avoid-automatically-updating-rails-timestamp-fields
> 
> —
> Reply to this email directly or view it on GitHub.
>